.storybook/manager-head.html

```diff
@@ -4,6 +4,7 @@
 <link
     rel="preload"
     href="https://fonts.googleapis.com/css?family=Inconsolata"
+    as="style"
```

Comment:
The browser was complaining that a `<link rel="preload"` needs an `as` attribute.